modules/utxo-lib/src/bitgo/wallet/chains.ts

```diff
@@ -0,0 +1,100 @@
+/**
+ * Defines BitGo mappings between bip32 derivation path and script type.
+ *
+ * The scripts for a BitGo wallet address are defined by their derivation path.
+ *
+ * The derivation path has the format `0/0/${chain}/${index}` (in rare cases the prefix is not 0/0)
+ *
+ * The address script type (ScriptType2Of3) is defined by the `chain` parameter.
+ *
+ * This file defines the mapping between chain parameter and address type.
+ */
+import { ScriptType2Of3 } from '../outputScripts';
+
+/**
+ * All valid chain codes
+ */
+export const chainCodes = [0, 1, 10, 11, 20, 21, 30, 31];
```

Comment:
This has me again wondering whether we should be inheriting this (as well as other types/constants below) from `Codes` in `unspents` - and I'm guessing this may be due to concerns over circular dependencies since `unspents` uses `utxo-lib`? If so the best solution might be to have the `Codes` types in a separate module. However that'd be out of scope for this PR. I'm just concerned that it's a bit confusing/harder to maintain with the codes defined in multiple places.

Reply:
I actually want this file to supersede `Codes` (I sort of mention it in the commit message). Let me add a deprecation notice to `@bitgo/unspents`.

Reply:
actually let's do this in a separate PR, ideally after https://github.com/BitGo/BitGoJS/pull/1765

Reply:
Alright that's fine with me, I'm not too concerned where it ends up, just that it only lives in one place. (and I'll try to read commit messages more closely in the future)